src/miner.cpp

```diff
@@ -764,23 +774,23 @@ void GenerateDynamics(int nCPUThreads, int nGPUThreads, const CChainParams& chai
         return;
     }
 
-    boost::thread_group* minerThreads = GetMinerThreads();
+    boost::thread_group* minerThreads = GetCPUMinerThreads();
```

Comment:
The problem was that we wouldn't have empty thread group here. Instead of keeping track of the threads we just remove them all, just like it was before.